client/packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -89,8 +97,15 @@ const SyncSettingsForm = ({
         component={
           <NumericTextInput
             value={intervalSeconds}
-            onChange={e =>
-              setSettings('intervalSeconds', Number(e.target.value))
+            onChange={seconds =>
```

Comment:
We don't have a `PositiveIntegerInput` component yet and I decided not to create one for a single usage. Maybe later